app/lib/domain_events/handlers/tier_change_handler.rb

```diff
@@ -0,0 +1,40 @@
+module DomainEvents
+  module Handlers
+    class TierChangeHandler
+      def handle(event)
+        return unless ENABLE_EVENT_BASED_PROBATION_CHANGE
+
+        Shoryuken::Logging.logger.info "event=domain_event_handle_start,domain_event_type=#{event.event_type},crn=#{event.crn_number}"
+        case_info = CaseInformation.find_by_crn(event.crn_number)
+
+        if case_info.nil?
+          Shoryuken::Logging.logger.error "event=domain_event_handle_failure,domain_event_type=#{event.event_type}," \
```

Comment:
I'm wondering if I should add a `logger` to the arguments of `def handle(...)` so we don't need to keep saying `Shoryuken::Logging.logger`...

Reply:
(Not asking for a change, just thinking out loud)